.github/workflows/CI.yml

```diff
@@ -0,0 +1,31 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: "3.6"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install -r requirements.txt -r test-requirements.txt -r docs-requirements.txt
```

Comment:
Should these use `python3` instead?

Reply:
The `setup-python` action will make `python` be whatever version was specified, so we don't have to do that. I probably would have gone with `python3` personally out of habit, but the github [docs](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python) I followed just used `python` so I stuck with that.